src/DbConnector.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace RobotStores;
+// FYI need '\PDO' in order to refer to global  namespace of PHP where PDO lives
+
+class DbConnector
+{
+    static private $db;
+
+    static public function getConnection(): \PDO
```

Comment:
do we need a \ in the typehint? no idea!

Reply:
we're not 100% sure, but we think so - it doesn't seem to be very happy if we take away the back slash. We need it in order to refer to the global namespace.

Reply:
cool happy to leave as is